src/session-adapters/cookieAdapter.ts

```diff
@@ -4,37 +4,48 @@ import { Adapter } from './generalAdapter'
 
 const clientSecret = process.env['CLIENT_SECRET'] || ''
 
+const createScopedKey = ({
+  spaceId,
+  userId,
+  key,
+}: {
+  spaceId: string
+  userId: string
+  key: string
+}) => {
+  return `${spaceId}:${userId}:${key}`
+}
+
+// We do not use `clientId` in cookie adapter,
+// because different plugins will have different domain names,
+// and it's enough to differentiate these cookie values.
```

Comment:
This is true, but what about DBs in future? There might be a shared database for all apps, in that case the app client id would be needed. WDYT?

Reply:
Ah yes! I added that comment to `cookieAdapter`. And as you suggested yesterday, I added `clientId` to `Adapter` type :D 

It's just that cookie adapter is not using clientId, but for supabase adapter, it can use it.

https://github.com/storyblok/app-extension-auth/blob/improve-session-store/src/session-adapters/generalAdapter.ts#L9